lib/module.ts

```diff
@@ -99,7 +99,7 @@ export class Module<
     return flattenedModule
   }
 
-  private getMetadata(): IFlattenedModuleMetadata {
+  private getMetadata(): IModuleMetadata {
```

Comment:
It seemed like the metadata here isn't necessarily specific to a flattened module, so I changed the name.